EveripediaNetwork/ep-ui
# fix editor opening up wrong wikis

# Summary
Fixes issue where the editor opens up the wrong wiki when editing a wiki because the wrong id was used to save a draft earlier.

### How to test
- Open a wiki in the editor, make edits
- Open another wiki in the editor
- It should open up the correct wiki
Fixes: https://github.com/EveripediaNetwork/issues/issues/3096
